### PR TITLE
Remove AB3 occurences and fix minor errors

### PR DESCRIPTION
**Pull request by yizhong187**
As per title.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 1. Download the latest `.jar` file from [here](https://github.com/se-edu/addressbook-level3/releases).
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+1. Copy the file to the folder you want to use as the _home folder_ for your DDD.
 
 1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar addressbook.jar` command to run the application.<br>
```
Comment (Teddayz) [inline docs/UserGuide.md:24]: > Perhaps java -jar addressbook.jar should be changed as well.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -172,13 +155,13 @@ Examples:
 
 ### Editing the data file
 
-AddressBook data are saved automatically as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
+DDD data are saved automatically as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
```
Comment (Teddayz) [inline docs/UserGuide.md:158]: > addressbook.json

**Review by Teddayz — COMMENTED** [review]
Overall LGTM!

**Review by FYL2003 — COMMENTED** [review]
LGTM

**Pull request merged**